src/compiler/program.ts

```diff
@@ -148,28 +175,6 @@ namespace ts {
             });
         }
 
-        function writeFile(fileName: string, data: string, writeByteOrderMark: boolean, onError?: (message: string) => void) {

```

Comment:
There is similar `writeFile` in `createCompilerHostFromProgramHost` that needs to be fixed as well..

Reply:
Why are there so many copies of this pattern?